src/screens/game-setup/GameSetup.tsx

```diff
@@ -0,0 +1,67 @@
+import React, {useState} from 'react';
+import { Button, SafeAreaView, Switch, Text, TextInput, View } from "react-native";
+
+const GameSetup = () => {
+  const [playerAName, setPlayerAName] = useState('');
+  const [playerBName, setPlayerBName] = useState('');
+  const [scoringMethod, setScoringMethod] = useState(false);
+  const [bestOfGames, setBestOfGames] = useState(false);
+  const [pointsPerGame, setPointsPerGame] = useState(false);
+
+  return (
+    <>
+      <SafeAreaView>
+        <View>
+          <TextInput
+            placeholder={'Player A Name'}
+            onChangeText={updatedName => {
+              setPlayerAName(updatedName);
+            }}
+            value={playerAName}
+          />
+          <TextInput
+            placeholder={'Player B Name'}
+            onChangeText={updatedName => {
+              setPlayerBName(updatedName);
+            }}
+            value={playerBName}
+          />
+        </View>
+        <View>
+          <Text>Scoring method</Text>
+          <View>
+            <Switch
+              value={scoringMethod}
+              onValueChange={newValue => {
+                setScoringMethod(newValue);
```

Comment:
Would be nice to see something like 
`setScoringMethod(Scoring.American)` or `setScoringMethod(Scoring.English)`